crowbar_framework/Gemfile

```diff
@@ -58,7 +58,7 @@ gem "rack-mini-profiler", "~> 0.9",
 
 group :development, :test do
   gem "brakeman", "~> 2.6"
-  gem "rspec-rails", "~> 2.14"
+  gem "rspec-rails", "~> 3.1"
```

Comment:
I would like to have the travis / developer environment be reasonably similar to the production environment to make sure that there are no uncaught accidental breakages.